static/app/components/sidebar/sidebarDropdown/switchOrganization.tsx

```diff
@@ -10,78 +10,113 @@ import {IconAdd, IconChevron} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import space from 'sentry/styles/space';
 import {OrganizationSummary} from 'sentry/types';
+import shouldUseLegacyRoute from 'sentry/utils/shouldUseLegacyRoute';
+import useOrganization from 'sentry/utils/useOrganization';
+import useResolveRoute from 'sentry/utils/useResolveRoute';
 import withOrganizations from 'sentry/utils/withOrganizations';
 
 import Divider from './divider.styled';
 
+function OrganizationMenuItem({organization}: {organization: OrganizationSummary}) {
+  const currentOrganization = useOrganization();
+  const {slug} = organization;
+
+  const menuItemProps: Partial<React.ComponentProps<typeof SidebarMenuItem>> = {};
+
+  const route = useResolveRoute(organization, `/organizations/${slug}/`);
+
+  if (!route) {
+    return null;
+  }
+
+  if (shouldUseLegacyRoute(organization)) {
+    if (currentOrganization.features.includes('customer-domains')) {
```

Comment:
Is this correct? Shouldn't using legacy routes mean that the organization doesn't have customer domains?